templates/candidate/candidate_list.html

```diff
@@ -328,6 +328,10 @@ <h1>Candidates</h1>
     <br />
 
     {# Turn on/off review mode #}
```

Comment:
Please move this "Turn on/off review mode" comment to be above "review_mode_id" code, and add a new comment here saying something like "Limit list to only show candidates with an email address".